moveit_ros/occupancy_map_monitor/src/occupancy_map_updater.cpp

```diff
@@ -39,6 +39,8 @@
 
 namespace occupancy_map_monitor
 {
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_ros.occupancy_map_updater");
```

Comment:
```suggestion
static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_ros_occupancy_map_updater");
```

Nit: @henningkayser Which one should we use .?